packages/edit-site/src/components/layout/index.js

```diff
@@ -282,7 +282,7 @@ export default function Layout() {
 							// (https://github.com/WordPress/gutenberg/pull/51558/files#r1231763003),
 							// so we can't remove the element entirely. Using `inert` will make
 							// it inaccessible to screen readers and keyboard navigation.
-							inert={ showSidebar ? undefined : 'inert' }
+							inert={ showSidebar ? undefined : 'true' }
```

Comment:
I think this is a typo because the value of the inert attribute is supposed to be [a boolean value](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/inert#value). Perhaps values other than `false` are implicitly treated as `true`.

Reply:
Thanks for saying it was a typo to cover me, but I believe that was my doing. I think I just got it wrong 😅 Nice catch!